care/facility/migrations/0102_auto_20200420_1116.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.11 on 2020-04-20 05:46
+
+import care.utils.models.jsonfield
+from django.db import migrations
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [third party import "from django.db import migrations" should be placed before "import care.utils.models.jsonfield"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5384758)